lib/site_prism/element_container.rb

```diff
@@ -130,9 +130,11 @@ def create_waiter(element_name, *find_args)
       create_helper_method method_name, *find_args do
         define_method method_name do |timeout = nil, *runtime_args|
           timeout = timeout.nil? ? Waiter.default_wait_time : timeout
-          Capybara.using_wait_time timeout do
+          result = Capybara.using_wait_time timeout do
             element_exists?(*find_args, *runtime_args)
           end
+          raise SitePrism::TimeOutWaitingForExistence if !result && SitePrism.raise_on_wait_fors
```

Comment:
as this is a new error, can we have them named with `Error` at the end. I'm aware the others don't but it's a breaking change to refactor them.